README.md

```diff
@@ -29,7 +29,7 @@ If you have any questions, queries or comments, [RocketChat] is the place to fin
 
 ## Besu Users
 
-Users wanting to install Besu should install the Besu binary by following [these instructions](https://besu.hyperledger.org/HowTo/Get-Started/Install-Binaries/).    
+Users want to install Besu should install the Besu binary by following [these instructions](https://besu.hyperledger.org/HowTo/Get-Started/Install-Binaries/).    
```

Comment:
```suggestion
To install the Besu binary, follow [these instructions](https://besu.hyperledger.org/HowTo/Get-Started/Install-Binaries/).    
```

Reply:
@saumyaagrawal2001 if you commit my suggestion above, and add a signoff in the commit message, we should be able to merge this.